config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -93,7 +93,7 @@ IF(PYTHONINTERP_FOUND)
     IF(Boost_FOUND)
         FIND_LIBRARY(Boost_PYTHON_LIBRARY
                      NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}   # Linux with multiple Python versions
-                           boost_python-mt                                                 # Mac
+                           boost_python${PYTHON_VERSION_MAJOR}                             # Mac with Python 3
```

Comment:
The convention `boost_python-mt` actually applies to our lab's test Mac, so it cannot be removed. Is there a system where `boost_python${PYTHON_VERSION_MAJOR}` is used, or is this just your suggestion of how the library should be named?